frontend/src/components/Navigator/components/SideNav.js

```diff
@@ -5,15 +5,17 @@
 */
 import React from 'react';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
What do you think about importing just specific functions here instead of all of lodash?

`import { startCase } from 'lodash';`